wandb/sdk/launch/builder/noop.py

```diff
@@ -51,5 +51,6 @@ def build_image(
         For this we raise a launch error since it can't build.
         """
         raise LaunchError(
-            "Attempted build with noop builder. Specify a builder in your launch config at ~/.config/wandb/launch-config.yaml"
+            "Attempted build with noop builder. Specify a builder in your launch config at ~/.config/wandb/launch-config.yaml.\n"
+            "Note: Jobs sourced from Docker images do not require a builder. See https://docs.wandb.ai/guides/launch/create-job."
```

Comment:
I might go even further and say Jobs sourced from git repos and code artifacts do require a builder

Reply:
Updated!